test/schema-tests.js

```diff
@@ -13,10 +13,14 @@ describe('convict schema file', function() {
     }
   });
 
-  it('must parse a config specification from a file', function() {
+  beforeEach(function() {
```

Comment:
It does mean that if the schema file can't be parsed, it'll fail all tests within `schema-tests.js`. I can restructure the tests if you think that's a problem not worth introducing.